packages/cli/src/watch-handler.ts

```diff
@@ -42,11 +46,11 @@ export class WatchHandler {
             this.options.diagnosticsManager ?? new DiagnosticsManager({ log: this.log });
     }
 
-    private listener: WatchEventListener = async (event) => {
+    public readonly listener = async (event: IWatchEvent) => {
```

Comment:
not sure the readonly does something here

Reply:
Typescript will emit `Cannot assign to 'listener' because it is a read-only property`.